modules/nixosConfigurations.linux-01/github-runner-multi-arch.nix

```diff
@@ -1,21 +1,26 @@
-{config, lib, ...}: {
+{ config, lib, magicPaths, ... }:
+let
+  githubRunnersCfg = {
+    count = 10;
+    namePrefix = "multi-arch";
+  };
+
+  mkList = prefix: (builtins.genList
+    (x: "${prefix}${githubRunnersCfg.namePrefix}-${builtins.toString x}")
+    githubRunnersCfg.count);
+
+in
```

Comment:
Might be a nit pick, but I think it's often better to keep abstractions in nixos configs at a minimum and only use it where it's really necessary.
Extra abstractions always come at a cost. They increases the mental capacity required to understand the code and make it harder to grasp what's happening, especially for less experienced nix users.